common/cryptfs.c

```diff
@@ -209,7 +215,7 @@ load_integrity_mapping_table(int fd, const char *real_blk_name, const char *meta
 			DEBUG("DM_TABLE_LOAD successfully returned %d", ioctl_ret);
 			break;
 		}
-		usleep(500000);
+		nanosleep(&ts, &ts);
```

Comment:
provide a wrapper functions or define a macro in common/macro.h

e.g.:
```#define` NANOSLEEP(nsec) \
   {	struct timespec ts;\
	ts.tv_sec = 0;\
	//half a second sleep\
	ts.tv_nsec = nsec;\
        nanosleep(&ts, &ts);\
    }
    ```

Reply:
Added a nanosleep macro and changed all nanosleep function calls to it.